src/routes/safe/components/Transactions/helpers/TxParametersDetail/index.tsx

```diff
@@ -36,6 +51,11 @@ const StyledButtonLink = styled(ButtonLink)`
   }
 `
 
+const StyledDivider = styled(Divider)`
+  margin-right: -16px;
```

Comment:
You could use here the values defined in `src/theme/variables.js`

Reply:
I'll adjust it now. Bear in mind that they are not [true theme variables](https://v4.mui.com/customization/spacing/) though.